.envrc

```diff
@@ -0,0 +1,2 @@
+eval "$(lorri direnv)"
```

Comment:
This will break for anybody who uses direnv without nix.

Reply:
True, if anyone uses direnv configurations that aren't part of this repo. If they do, they still have a choice of not enabling this `.envrc` with `direnv allow`. Direnv configuration is opt-in for exactly this reason.

However, I'm not insisting that any of the above should be included in the repo. I'm simply sharing mine, with the idea in mind that others might find it useful too.